api/server/sdk/cluster_test.go

```diff
@@ -291,3 +293,105 @@ func TestSdkInspectIdNotPassed(t *testing.T) {
 	assert.Equal(t, serverError.Code(), codes.InvalidArgument)
 	assert.Contains(t, serverError.Message(), "Node")
 }
+
+func TestSdkAlertEnumerate(t *testing.T) {
```

Comment:
what is this test testing ?

Reply:
It is testing that the grpc connection goes to the server, then the server calls the interface, and returns the expected value.